### ISSUE TITLE
swagger module should support JAXBElement

### ISSUE THREAD

**Issue by sakthisunda** [stoicflame/enunciate#885]
Enunciate version 10.x - 11.x
JAX-RS web api 

In all versions it produces wrong Response class unless the return value is a Simple Type.
If it is generic type like the following examples, it generates the following info (viewing from Enunciate generated swagger ui )

Response Class (Status 200)
file

Some of the non-working signatures:  

```java
    public JAXBElement \<GroupType> getGroup(@PathParam("id") final String groupId, @Context final UriInfo uriInfo) throws PresentationException
```

```java
    public JAXBElement \<VXExtensionType> getExtension(@Context final UriInfo uriInfo) 
```

```java
    public Response \<ContainerType> getResourceClasses(final @Context HttpServletRequest request ..)
```

```java
    public JAXBElement \<MetadataType> getCXMetadata(@Context final UriInfo uriInfo, @PathParam("id") final String id) {
```

Question:

1. Will these work only if I provde @returnWrapped annotation ? 
2. Any future version that this will be supported without updating the code without @returnWrapped ?

NOTE: All the referred classes are available in classpath. When I use @returnWrapped with full path of the Class, they generate correct documentation. I tried atleast for one of them.

**Comment by sakthisunda** [comment]
Will this fix also be available in enunciate-openapi module generated openapi.yaml ?

**Comment by stoicflame** [comment]
Started digging into this, but I need some more context so I can correctly implement an enhancement.

Is `JAXBElement` intended to be used for both XML and JSON endpoints?

If intended to be used in JSON endpoints, how is the result expected to be the unwrapped value? Do you have some kind of custom Jackson mapping logic for `JAXBElement`? Jackson, by default, doesn't "unwrap" the value out of the `JAXBElement` when it serializes the data.

**Comment by stoicflame** [comment]
Moving to release 2.13, pending response with more information.

**Comment by sakthisunda** [comment]
> Started digging into this, but I need some more context so I can correctly implement an enhancement.
> 
> Is `JAXBElement` intended to be used for both XML and JSON endpoints?
> 
> If intended to be used in JSON endpoints, how is the result expected to be the unwrapped value? Do you have some kind of custom Jackson mapping logic for `JAXBElement`? Jackson, by default, doesn't "unwrap" the value out of the `JAXBElement` when it serializes the data.

For now, it is intended for only xml end point. I mean accept-type : application/xml

**Comment by stoicflame** [comment]
Fix to be included in 2.13.

**Comment by stoicflame** [comment]
[Enunciate 2.13.0 has been released.](https://github.com/stoicflame/enunciate/releases/tag/v2.13.0)
